.github/workflows/frontier-rpc-tests.yml

```diff
@@ -0,0 +1,54 @@
+name: Frontier RPC Tests
+on:
+  workflow_dispatch:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Comment:
Isn't this a bit too much though for the current resources the CI has?
Perhaps triggering it on PR merge or running it as part of the release build?

I'm primarily asking this since the CI will be very much prolonged since the entire binary has to be built.

Reply:
Agree, maybe we can also have a PR comment to trigger the rpc tests like benchmarks

Reply:
this is using github runner but yes makes sense, will trigger only on merge

Reply:
> Agree, maybe we can also have a PR comment to trigger the rpc tests like benchmarks

you can manually trigger it